platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/VirtualMapBaseBench.java

```diff
@@ -227,6 +257,50 @@ protected void verifyMap(long[] map, VirtualMap<BenchmarkKey, BenchmarkValue> vi
         }
     }
 
+    protected List<VirtualMap<BenchmarkKey, BenchmarkValue>> saveMaps(
```

Comment:
I wonder if the existing `saveMap()` method may be used instead of adding this new method?

Reply:
No, and this is a part of the reason why the fix took a bit of time. I tried using the existing method and after a couple of days of trying synced up with Oleg, spent some time debugging it, and figured it's designed to save a single map only. The ReconnectBench needs two maps, and the existing method would save them separately. This would cause the existing restoreMap() logic to create a MerkleDb that only knows about the first map being restored and refusing to restore the second map.

After evaluating options, I decided to implement the new `saveMaps` method that allows one to save several maps and be able to restore them using the existing code.

We could consider removing the legacy `saveMap` at a later time because the new one can replace it.